beta-src/src/components/WDActionIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import { useState } from "react";
+import { SvgIcon, Box } from "@mui/material";
+import { ActionIcon, ActionIconSelected } from "./svgr-components";
+
+const WDActionIcon: React.FC = function () {
+  const [active, setActive] = useState(false);
+
```

Comment:
const isActive = active === true;

Reply:
For this change, are you asking that I change the default state to "true"? Would it be better to keep it false so that the button isn't immediately selected on component render?

Reply:
For this change, is it asking it me to change the default state of active to "true"? I set it to false initially so that the icon isn't automatically selected on component render.

Reply:
no, it's saying isActive is the same value as active === true. so if active is false, then active === true will be false.